ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -77,6 +77,16 @@ type structure_rule = {
     "table" ? network_valid_routing_table
     @{priority, The priority of the rule over the others. Required by Network Manager when setting routing rules.}
     "priority" ? long(0..0xffffffff)
+    @{nmstate-action used by nmstate module}
```

Comment:
i am not in favour of the naming.
at least make a `structure_rule_nmstate` and move them there. 

are these simply ignored by the old network.pm code? if so, i see not much harm.

wrt separate scheam's i can make a PR. i need to work in the core schema anyway to add the IB support.

Reply:
@stdweird I may need some help/guide from you to move this into the new structure you created, please.